ceaserCipher/ceaserCipher.js

```diff
@@ -0,0 +1,25 @@
+const ceaserCipher = (string, key) => {
```

Comment:
[OPTIONAL]
It is actually spelled caesar

Reply:
Thanks @mcrd25